libraries/AP_Scripting/applets/Aerobatics/FixedWing/plane_aerobatics.lua

```diff
@@ -2173,6 +2174,11 @@ function do_path()
       last_named_float_t = now
       gcs:send_named_float("PERR", pos_error_ef:length())
    end
+   
+   if ((current_measured_pos_ef - path_var.start_pos):z()) > AEROM_ALT_ABORT.get() then
```

Comment:
this is supposed to be ```:get()```, right?

Reply:
I think maybe the > is backwards?
Also, this is probably a lot easier to think about if it was this:
```suggestion
   if ((current_measured_pos_ef:z() - path_var.start_pos:z()) < AEROM_ALT_ABORT:get() then
```

Reply:
right...but I was using the wrong variables also...fixed and tested now